herddb-core/src/main/java/herddb/model/Transaction.java

```diff
@@ -509,7 +507,7 @@ public boolean isOnTable(String name) {
                 || (newTables != null && newTables.containsKey(name));
     }
 
-    public void synch() throws LogNotAvailableException {
+    public void sync() throws LogNotAvailableException {
```

Comment:
Is this one useful?

Reply:
Really for.. a test

Reply:
Please add //visible for testing

Reply:
I'll add it... (I'll check if I can reduce visibility too)